src/com/goide/psi/impl/GoPsiImplUtil.java

```diff
@@ -1130,6 +1130,15 @@ public static boolean isCImport(@NotNull GoImportSpec importSpec) {
     return GoConstants.C_PATH.equals(importSpec.getPath());
   }
 
+  public static boolean isRelativeImport(@NotNull GoImportSpec importSpec) {
+    return GoImportReferenceSet.isRelativeImport(importSpec.getPath());
```

Comment:
The idea was to add the method to the ImportSpec so that it's usable in that context (and to ask if I should move the `GoImportReferenceSet.isRelativeImport` to `GoPsiUtil`. I guess it's not a good idea? :D